README.md

```diff
@@ -0,0 +1,182 @@
+> - Allison
+>   Well, that is a nice side effect. The main thing is that because of the
+>   trademark relationship, there's a few places where we really *can't* use
+>   the Camel at all. Say, if we published a collection of Perl documentation
+>   or something like that, we really could not use the Camel in that context.
+>   Because that's the one that O'Reilly is protecting it for.
+>   So we started the process just to give us another option in a few places.
+>   But as we go on it seems to have kind of taken a life of its own. It's been
+>   showing up on YAPC T-shirts and things like that.
+> - Philippe
+>   It's a nice logo, because the onion has been part of Larry's talks for a
+>   while, so it's already part of the culture.
+> - Allison
+>   Yeah. We spent a good deal of time looking for the right logo. We tried a
+>   whole bunch of different things. At the very beginning we thought "Not an
+>   onion, because it doesn't make a very good logo." We looked for something
+>   else and just kept coming back to the onion because of that connection to
+>   Perl history. Then we found an artist who did an onion that we liked so we
+>   went with it.
+> - Who's the author of the onion?
+>   His name is Devin Shane Muldoon. He's an artist in the San Francisco area.
+>   He's really talented.
+
+A [French translation of the
+interview](https://articles.mongueurs.net/interviews/randal-2005.html) was
+published in GNU/Linux Magazine France in 2006.
+
+## See also
+
+- <https://dev.to/thibaultduponchelle/the-perl-complex-topic-of-logos-3161>
+- <https://dev.to/smonff/notes-about-the-ongoing-perl-logo-discussion-4c80>
+- <https://youtu.be/areXIy7UKyA?si=IRyarBguLwCuBXtd&t=1701>
+
+## FAQ
+
+- Will the logos change?
+
+  We currently consider this artwork to be production ready, but also a work in
+  progress. We may alter the logos moving forward if we see a need, but we
+  don't imagine radical changes to be forthcoming.
+
+- Who put you in charge?
+
+  Nobody. Traditionally things happen in Perl because somebody does something
+  and enough other people think it's worth adopting. It's in this spirit that
+  the logo is being presented. Our hope is that all (or most) of us can
+  standardize on the same artwork. This current working group arose out of a
+  conversation at the 2024 Perl Toolchain Summit in Lisbon. We felt we had
+  enough stakeholders in the same room to come to an agreement in principle and
+  then move ahead with the project.
+
+- Can I Put This on a T-shirt, Mug or Candle?
+
+  Please do! We understand that attribution may not be possible in these
+  circumstances. In these cases we would rather you use the logo than worry
+  about the attribution aspect of the license. You can be creative as well. If,
+  for example, the t-shirts are for a conference, you could add the attribution
+  to the conference website. At the end of the day, it's your logo and it's
+  your call as to whether you can reasonably add the attribution.
```

Comment:
This logo is not being registered as a trademark, nor being used as an unregistered trademark (which is also a "thing" in some jurisdictions). The creativecommons warning is for scenarios where trademark-ish status is sought. I don't see that's the case here. @oalders Am I right on that?

In my view, the wording "not in any way that suggests the licensor endorses you or your use" solves any plausible problem, and implicitly disallows trademark registration by some third party. I cannot imagine a court would allow that to stand.

Reply:
I felt it worth adding: nothing I've said here constitutes legal advice, which means if I'm wrong, you can't sue me. If you pay a lawyer, you engage their professional insurance, and then if they're wrong you *can* sue them. If you have real concerns, it's probably only a couple of hundred GBP to get advice. My feeling here is it's not worth it, but see the start of this paragraph!